src/processes/follow-topic-process.js

```diff
@@ -131,9 +130,13 @@ const followTopicProcess = async (job, done) => {
           members: [communityMessageFormat.user_id, data.user_id],
           created_by_id: communityMessageFormat.user_id
         });
-
-        await chat.create();
+        try {
+          await chat.create();
+        } catch (error) {
+          console.log('::: Error on create chat :::', error);
+        }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding error handling to the `chat.create()` method call is a good practice. It prevents unhandled promise rejections, which can lead to unexpected behavior or crashes. However, simply logging the error might not be sufficient in a production environment. Consider throwing the error after logging it, or handling it in a way that's appropriate for your application's error strategy.
```diff
- await chat.create();
+ try {
+   await chat.create();
+ } catch (error) {
+   console.log('::: Error on create chat :::', error);
+ }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->